docs/how-to/service-dependencies.md

```diff
@@ -1,11 +1,297 @@
 # How to manage service dependencies
 
-Pebble takes service dependencies into account when starting and stopping services. When Pebble starts a service, it also starts the services which that service depends on (configured with `required`). Conversely, when stopping a service, Pebble also stops services which depend on that service.
+If you are using Pebble to manage your services, chances are, you've got more than one service to manage.
+
+And when orchestrating more services, things can and will get trickier, because more often than not, those services depend on each other to function together, and managing dependencies is no mean feat.
+
+Let's have a look at a concrete example of service dependencies.
```

Comment:
I suggest removing this and providing more context instead. Similar to the feedback provided about the tutorial.

E.g.

This document shows how to manage a set of services that have dependencies on each other to function properly. A simple web application is used to demonstrate the instructions in this guide.